app/src/main/java/me/thuanc177/kotsune/libs/animeProvider/hianime/HiAnimeExtractor.kt

```
@@ -0,0 +1,275 @@
+
+                    extractedData["outro"] = srcsData.optJSONObject("outro")?.let {
+                        mapOf("start" to it.optInt("start", 0), "end" to it.optInt("end", 0))
+                    } ?: mapOf("start" to 0, "end" to 0)
+
+                    extractedData["tracks"] = srcsData.optJSONArray("tracks")?.let {
+                        MutableList(it.length()) { i -> it.getJSONObject(i).toMap() }
+                    } ?: emptyList<Any>()
+
+                    extractedData["sources"] = sources
+                    return extractedData
+                } catch (e: Exception) {
```

detekt (1.23.7)

[warning] The caught exception is swallowed. The original exception could be lost.

(detekt.exceptions.SwallowedException)